internal/api/instrumentation/probes.go

```diff
@@ -15,6 +15,7 @@ import (
 
 const (
 	labelDb                    = "db"
+	labelPlaybookRunCancel     = "playbook_run_cancel"
```

Comment:
this label value does not seem to be used anywhere, except for the counter initialization